docs/references/general/rawacf.md

```diff
@@ -84,11 +77,10 @@ RAWACF files contain a record that contains scalar and vector fields.
 | *rawacf.revision.minor* | *None*   | int       | Minor version number of the RAWACF algorithm                                            |
 | *thr*                   | *None*   | float     | Threshold factor                                                                        |
 
-
-### Vectors 
-
 !!! Note
-    *slist* contains the range gates that obtained data points during the integration period of the beam. 
+    The original intended usage of the `channel` parameter was to identify the data from the two separate frequency channels of the stereo radars (e.g. Hankasalmi). However, this parameter has sometimes been used for non-stereo radars when data from two different frequencies have been recorded in a single file.
```

Comment:
```suggestion
    The original intended usage of the `channel` parameter was to identify the data from the two channels of the stereo radars (e.g. Hankasalmi). However, this parameter has sometimes been used for non-stereo radars to identify data sampled using a different frequency, beam pattern, control program, or other operating parameter, and then provided in a single file.
```

Reply:
Thanks for making these edits @ecbland! I think the first change (removing "separate frequency") sounds good, but the latter change about non-stereo radars may be unnecessary. I suppose the difference here is it's discussing the internal channel identifier rather than the "external" one in the filename?

Reply:
Although I'm probably confusing the new Borealis multi-file twofsound data with the old single-file ROS data, so nevermind!

Reply:
Sure - sorry, that's my fault for trying to write a reply too early and on too little sleep!

Reply:
I actually wonder if this "Note" is even necessary, given that the Stereo vs multi-frequency usage is already explained in channel field of the above table.